### PR TITLE
Murky: new recipe

### PR DESCRIPTION
**Pull request by prjemian**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by prjemian** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes : Ready for review when you have a chance. Thanks in advance!

**Comment by prjemian** [comment]
@conda-forge/help-python : This is ready for review, when you get a chance.  Thanks!

**Comment by prjemian** [comment]
@conda-forge/staged-recipes : This is ready for review, when you get a chance. Thanks!

File: recipes/murky/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/murky-{{ version }}.tar.gz
+  sha256: 6060d31114bd116dd5a5f1575e1cacdb06e7e4e8877405922300ee8bf101b780
+
+build:
+  skip: true  # [py<3.8, py>=311]
```
Comment (ocefpaf) [inline recipes/murky/meta.yaml:13]:
```suggestion
  skip: true  # [py<3.8 or py>=311]
```

**Comment by prjemian** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes : I revised the source (added pyyaml package requirement and expanded the Python versions to include 3.7) and released a new tag that passed here.

**Pull request merged**